WMFModel/WMFArticle+CoreDataClass.h

```diff
@@ -0,0 +1,12 @@
+#import <Foundation/Foundation.h>
+#import <CoreData/CoreData.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface WMFArticle : NSManagedObject
```

Comment:
Any reason why the class file is not "WMFArticle.h"? 

Haven't set up a stack from the ground up in a while. Is this new convention?